scio-core/src/main/scala/com/spotify/scio/ScioContext.scala

```diff
@@ -86,7 +86,7 @@ object ScioContext {
   /** Create a new [[ScioContext]] instance for testing. */
   def forTest(): ScioContext = {
```

Comment:
Would it make sense to change the signature here and expect `Class name` here. There would be only one way to get `testId`?